runtime/onert/backend/train/ops/PoolLayer.cc

```diff
@@ -52,8 +52,8 @@ class MaxPool2D final : public TrainingKernelRegistry
   const IPortableTensor *_output;
   nnfw::cker::PoolParams _op_params;
 
-  std::unique_ptr<Tensor> _act_back_prop_output;
-  std::unique_ptr<Tensor> _arg_max_index;
+  std::unique_ptr<Tensor> _act_back_prop_output = nullptr;
+  std::unique_ptr<Tensor> _arg_max_index = nullptr;
```

Comment:
This is not caused by Coverity.

While reading the code, I just found out it is better to assign `nullptr` explicitly.

Reply:
I first thought it is better to assign `nullptr` "explicitly", although `unique_ptr` has a default null value.
I thought it might help read the code, a little bit. 

On second thought, "`unique_ptr` has null value on default" is well-known.
So, Without explicit assigning also looks good for me.

Better to remove explicit assigning to focus on the uinitialization-issue.

Reply:
(suggestion)
How about using the initializer list for consistency with other class member?

Reply:
> How about using the initializer list for consistency with other class member? 

Thanks for the suggestion! 

But I changed the mind and decided not to handle the un-initialization issue in PoolLayer. 
  * Because Initialization in PoolParams(like https://github.com/Samsung/ONE/pull/12036) looks enough. So Duplicated-initialization here is a bit redundant. 

So, I closed this PR. Thanks for reviewing.